### PR TITLE
Add .gitattributes and .gitignore to crane init Template

### PR DESCRIPTION
**Pull request by mrdavidlaing**
In line with Crane's `Build you into the pit of success` mantra; I'd like to propose that the project skeleton includes `.gitignore` and `.gitattributes` files that properly customise git behaviour for Windows / VS

### CODE REVIEW

**Comment by mrdavidlaing** [comment]
@ewilde, @kevholditch,

I think I've put the files in the right place, but I'm unsure how to write tests for this behaviour.

Could you possible show me how by committing some test stubs to the branch for me to complete.

Thanks!

File: src/Crane.Integration.Tests/Features/Templates/VisualStudioTemplateFeature.cs [hunk 1]
```diff
@@ -53,11 +53,14 @@ public void creating_a_visual_studio_setup_using_the_template(DirectoryInfo root
...
             "It should replace the tokens in the nuget specification file"
-                ._(() =>File.ReadAllText(Path.Combine(context.SourceDirectory.FullName, "ServiceStack", string.Format("{0}.nuspec", "ServiceStack")))
+                ._(() => File.ReadAllText(Path.Combine(context.SourceDirectory.FullName, "ServiceStack", string.Format("{0}.nuspec", "ServiceStack")))
                     .Should().NotContain("%username%")
                     .And.NotContain("%context.ProjectName%")
                     .And.NotContain("%DateTime.Now.Year%"))
                 .Teardown(() => Directory.Delete(context.ProjectRootDirectory.FullName, recursive: true));
+
+            "It should create a .gitignore file in the project root"
+                ._(() => File.Exists(Path.Combine(context.SourceDirectory.FullName, ".gitignore")));
```
Comment (ewilde) [inline src/Crane.Integration.Tests/Features/Templates/VisualStudioTemplateFeature.cs]: Example of how to test a file gets deployed in a template

**Comment by ewilde** [comment]
@mrdavidlaing thanks for the contrib, great spot! We also forgot to mention that in our contribution workflow we push to develop first and then got from develop to master at the end of a milestone. (we then causes an official choco package to get built and deployed by teamcity). I've raise #88 to address this

**Comment by mrdavidlaing** [comment]
@ewilde - Thanks for the test pointers.

Since it appears that github won't be [adding the feature to change the target branch of a PR anytime soon](https://github.com/isaacs/github/issues/18), I'm going to close this PR and open a new one targeting the `develop` branch.

**Pull request closed without merging**